schema/tables/tcc_access.yaml

```
@@ -0,0 +1,46 @@
+name: tcc_access
+platforms:
+  - darwin
+description: Information about macOS TCC database contents, for determining permissions granted to applications.
+columns:
+  - name: source
+    type: text
+    required: true
+    description: Either "user" or "system".
+  - name: service
+    type: text
+    required: true
+    description: The name of the TCC service.
+  - name: client
+    type: text
+    required: true
+    description: The bundle identifier or absolute path to the program using the TCC service.
+  - name: client_type
+    type: integer
+    required: true
+    description: Indicates whether client is a bundle identifier (0) or absolute path (1). 
+  - name: auth_value
+    type: integer
+    required: true
+    description: 'Indicates whether the access is: denied (0), unknown (1), allowed (2), or limited (3).'
+  - name: auth_reason
+    type: integer
+    required: true
+    description: TODO
+  - name: last_modified
+    type: bigint
+    required: true
+    description: The last time the entry was modified, in epoch seconds.
+  - name: policy_id
+    type: integer
+    required: false
+    description: The MDM policy that allows TCC access for the application.
+  - name: indirect_object_identifier
+    type: string
```

@rachaelshaw is this different than text?

---

this should be `text`
```suggestion
    type: text
```